pavement.py

```diff
@@ -30,7 +30,7 @@
 
 master_app = 'runestone'
 serving_dir = "./build/cpp4python"
-dynamic_pages = True
```

Comment:
When you submit a PR do not include changes to pavement.py.  Please change this back to True and push again.  I understand you need to have this False for your local development, but just don't `git add` pavement.py when you are getting a PR ready.

Thanks